### DISCUSSION TITLE
AlphaTex - is it possible to leave arbitrary text note on top of notes (alongside chords)?

### DISCUSSION THREAD

**Discussion by bali182** [CoderLine/alphaTab#1559] [Q&A] [answered]
I'd like to demonstrate some basslines using AlphaTab. I'd like to mark the chords alongside chord tones above certain notes.

In an example, is it possible to add arbitrary text for each note here on top of the chord symbols?

```ts
const content = `
\\title "Bassline sample"
\\instrument "AcousticBass"
.
\\clef Bass 0.4.4 {ch "Em"} 2.4.4 3.4.4 2.3.4  
      | 0.3.4 {ch "Am"} 3.3.4 3.4.4 2.4.4 
      | 0.4.1 {ch "Em"}
`
```

And this would be my goal:
![Screenshot 2024-06-27 145646](https://github.com/CoderLine/alphaTab/assets/3879181/2245ce20-b0b6-4b75-9676-dd7bf88d1647)

**Comment by Danielku15** [comment, marked as answer]
Adding any custom text (beat text) via alphaTex is covered in https://github.com/CoderLine/alphaTab/issues/1484

**Reply by bali182** [reply]
Looks like I just have to bump to 1.4, thanks!
